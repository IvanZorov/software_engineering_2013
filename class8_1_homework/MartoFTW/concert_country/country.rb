require 'csv'

all = []
CSV.foreach(ARGV[0]) do |row|
        if (row[2].to_s.downcase == ARGV[1].downcase)
                all << row
        end 
end
all = all.sort_by{|a,b,c,d,e,f| a}
CSV.open("country_result.csv","w") do |csv|
        all.each do |row|
                csv << row
	end
end


