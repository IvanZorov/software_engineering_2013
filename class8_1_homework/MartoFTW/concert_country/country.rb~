require 'csv'

all = []
CSV.foreach(ARGV[0]) do |row|
        if (row[2].to_s == ARGV[1])
                all << row
        end 
end
all = all.sort_by{|a,b,c,d,e,f| a}
CSV.open("country_expected.csv","w") do |csv|
        all.each do |row|
                csv << row
	end
end


