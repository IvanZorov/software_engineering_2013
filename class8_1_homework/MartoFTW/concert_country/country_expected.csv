Bob Marley,Reggae,Bulgaria,15,pork Pie,17/01/2012
Bon Jovi,Rock,Bulgaria,16,Vasil Levski,10/10/2010
Brother Ali,Metal,Bulgaria,80,Vasil Levski,18/03/2014
Cookie Monster,Reggae,Bulgaria,15,Pork Pie,17/01/2012
Fintroll,Metal,Bulgaria,80,Vasil Levski,18/03/2014
Hiltop Hoods,Reggae,Bulgaria,30	Pork Pie,17/01/2012
Jaya,Reggae,Bulgaria,16,Pork Pie,27/01/2012
Klek,Metal,Bulgaria,80,Vasil Levski,17/12/2012
Korpiklaani,Metal,Bulgaria,80,Vasil Levski,18/03/2014
Kreator,Metal,Bulgaria,40,Vasil Levski,18/03/2014
Moita Grupa,Rock,Bulgaria,60,Vasil Levski,16/09/2011
Overkill,Rock,Bulgaria,60,Vasil Levski,10/10/2010
Scorpions,Rock,Bulgaria,60,Vasil Levski,18/03/2014
Sodom,Reggae,Bulgaria,15,Pork Pie,17/01/2012
Trollhammer,Reggae,Bulgaria,15,Pork Pie,17/01/2012
Tumor,Reggae,Bulgaria,15,Pork Pie,15/12/2016
Whitesnake,Reggae,Bulgaria,15,Pork Pie,17/01/2012
